src/main/java/com/example/crm_gym/controllers/LoginController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.crm_gym.controllers;
+
+import com.example.crm_gym.services.UserService;
+import io.swagger.annotations.Api;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@Slf4j
+@RequestMapping("/user")
+@Api(produces = "application/json", value = "Operations for login and updating user's password in the application")
+public class LoginController {
+
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("/login")
+    public ResponseEntity<Map<String, String>> loginUser(
+            @RequestParam("username") String username,
+            @RequestParam("password") String password) {
+
```

Comment:
You will need change it to POST method . It is good to pass password in the body